--- gmerlin-1.0.0/icons/Makefile.am.orig	2010-11-09 20:06:32.000000000 +0100
+++ gmerlin-1.0.0/icons/Makefile.am	2011-07-25 18:30:44.349550758 +0200
@@ -109,14 +109,14 @@
 
 install-data-hook:
 	test -z $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps || $(mkdir_p) $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps
-	ln -sf $(DESTDIR)$(pkgdatadir)/mixer_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-alsamixer.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/player_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-player.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/transcoder_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-transcoder.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/camelot_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-camelot.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/kbd_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-kbd.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/plugincfg_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-plugincfg.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/recorder_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-recorder.png
-	ln -sf $(DESTDIR)$(pkgdatadir)/visualizer_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-visualizer.png
+	ln -sf $(pkgdatadir)/mixer_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-alsamixer.png
+	ln -sf $(pkgdatadir)/player_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-player.png
+	ln -sf $(pkgdatadir)/transcoder_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-transcoder.png
+	ln -sf $(pkgdatadir)/camelot_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-camelot.png
+	ln -sf $(pkgdatadir)/kbd_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-kbd.png
+	ln -sf $(pkgdatadir)/plugincfg_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-plugincfg.png
+	ln -sf $(pkgdatadir)/recorder_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-recorder.png
+	ln -sf $(pkgdatadir)/visualizer_icon.png $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-visualizer.png
 
 uninstall-local:
 	-rm -rf $(DESTDIR)$(prefix)/share/icons/hicolor/48x48/apps/gmerlin-alsamixer.png
