--- gmerlin-1.0.0/lib/Makefile.am.orig	2010-09-30 15:59:04.000000000 +0200
+++ gmerlin-1.0.0/lib/Makefile.am	2011-07-23 10:33:40.372806341 +0200
@@ -49,7 +49,7 @@
 bin_PROGRAMS = gmerlin_visualizer_slave
 
 gmerlin_visualizer_slave_SOURCES = visualize_slave.c
-gmerlin_visualizer_slave_LDADD = libgmerlin.la
+gmerlin_visualizer_slave_LDADD = libgmerlin.la -lpthread -ldl
 
 lib_LTLIBRARIES = libgmerlin.la
 libgmerlin_la_SOURCES = \
--- gmerlin-1.0.0/configure.ac.orig	2011-01-05 20:45:01.000000000 +0100
+++ gmerlin-1.0.0/configure.ac	2011-07-23 10:40:14.012819521 +0200
@@ -154,7 +154,7 @@
 PKG_CHECK_MODULES(FREETYPE, freetype2 >= $FREETYPE_REQUIRED, , AC_MSG_ERROR("freetype2 not found"))
 
 ICONV_LIBS=
-AC_CHECK_LIB(iconv, libiconv_close, ICONV_LIBS="-liconv")
+AC_CHECK_FUNC([iconv_close], , [AC_CHECK_LIB(iconv, libiconv_close, ICONV_LIBS="-liconv")])
 AC_SUBST(ICONV_LIBS)
 
 dnl
